### ISSUE TITLE
General Direction / Differences Between v1 and v2?

### ISSUE THREAD

**Issue by tonyb983** [edvin/tornadofx2#16]
*If an issue is the incorrect place for this please let me know and I'll close it.*

Is there a general **"design doc"** for this project? Something that explains what v2 hopes to accomplish over v1? What changes are in progress? What changes are complete? What do you guys need help with? Etc. etc. etc.

###  <3 Tornadofx, you guys rock, I'd love to help in whatever miniscule way I can / am able.

**Comment by aleksandar-stefanovic** [comment]
I agree with the sentiment, I think that many people are eager to contribute to this repository, but the work to be done is undefined. I think that this repository would get volunteer activity if someone was there to manage things. My suggestion would be to:

- enable Discussions on the repository (a new GitHub feature) — this would help the engagement with the commnunity
- state the general direction and the vision of the project
- assign a maintainer (maybe someone who has contributed on v1 and who you trust) that would watch over the repository and put the plan into action
- owner or maintainer should then create projects which describe general areas to be worked on, which should contain issues that are broken-down enough so that the community can easily contribute

I, for one, would gladly help with issue triaging (issue tagging, issue confirmation, pull request review etc.) if needed.

**Comment by SKeeneCode** [comment]
> I agree with the sentiment, I think that many people are eager to contribute to this repository, but the work to be done is undefined.

I think the thumbs up on your post reinforces this point. People want this project to continue but [it needs maintainers](https://github.com/edvin/tornadofx2/issues/21#issuecomment-807214024). 

I am not experienced enough to maintain something like this alone, but if a handful of others were willing to becoming a new set of maintainers and come together and agree on direction and work for the project I would happily throw my hat in the ring.

Would anyone else be interested? If we gather a few people, sit down and hash some stuff out we might be able to get into a position to do something.

**Comment by JacekWicka** [comment]
@SKeeneCode  I would like to help with maintaining the project though my time is also limited.

**Comment by tonyb983** [comment]
Sign me up, I'd love to help in whatever (probably minor) way I can.

**Comment by ctadlock** [comment]
Sorry Ive been away for a while; had to focus on my business during COVID. We are back working on our software that needs this project so will be more active in it.
